api/src/Modules/concom/Controller/BaseConcom.php

```diff
@@ -21,11 +21,75 @@ public function __construct(Container $container)
     }
 
 
-    protected function buildEntry(Request $request, $dept, $member, $note, $position)
+    protected function getConComPosition($account, $event = null)
+    {
+        if ($event == null) {
+            $event = \current_eventID();
+            if ($event === null) {
+                return array();
+            }
+        }
+        $sql = <<<SQL
+            SELECT
```

Comment:
I was about to ask if I could convince you to start using Atlas\Query classes, but while I'm sure I could write this query with Atlas, I don't know exactly how I'd do it, yet! 😄 

That said, I'd appreciate it if you'd start using it for more common queries (http://atlasphp.io/cassini/query/ , and lots of examples now in Stores API and the new test_prereqs). I want to start getting out of the habit of writing raw SQL as much as possible.